### PR TITLE
Move the authorization.proto and parts under /pathz

### PR DESCRIPTION
**Pull request by morrowc**
Move the authorization proto definition under the /pathz location.

Following up on offline conversation about moving this definition under it's controlling rpc defintion.

### CODE REVIEW

File: pathz/authorization/authorization.proto [hunk 1]
```diff
@@ -37,11 +37,11 @@
 //
 syntax = "proto3";
 
-package gnsi.authorization;
```
Comment (marcushines) [inline pathz/authorization/authorization.proto:40]: you don't need to namespace this the authorization proto can just reside in gnsi.pathz

File: pathz/authorization/authorization.proto [hunk 2]
```diff
@@ -37,11 +37,11 @@
...
+package gnsi.pathz.authorization;
 
 import "github.com/openconfig/gnmi/proto/gnmi/gnmi.proto";
 
-option go_package = "github.com/openconfig/gnsi/authorization";
+option go_package = "github.com/openconfig/gnsi/pathz/authorization";
```
Comment (marcushines) [inline pathz/authorization/authorization.proto:44]: no need for seperate package

Comment (tomek-US) [reply pathz/authorization/authorization.proto:44]: +1

File: pathz/authorization/authorization.proto (hunk 1 above)
Comment (morrowc) [reply pathz/authorization/authorization.proto:40]: done. I just removed this package line entirely.
and the upper directory pathz.proto imports this proto... into pathz.

File: pathz/authorization/authorization.proto (hunk 2 above)
Comment (morrowc) [reply pathz/authorization/authorization.proto]: removed this line.

**Comment by morrowc** [comment]
this is ready to merge?

**Comment by tomek-US** [comment]
Hmm. I thought that this PR will be abandoned as PR #30 has moved the authorization.proto directly to pathz directory. Did I miss soemthing?

**Comment by morrowc** [comment]
> Hmm. I thought that this PR will be abandoned as PR #30 has moved the authorization.proto directly to pathz directory. Did I miss soemthing?


you are not missing something :) I was, I believe!! 
I think there's some re-organization of the README parts in the respository still to do, but the proto bits should
now be all sorted out.

**Comment by morrowc** [comment]
Ok, I think at this point there's only 2 files changed:
  README.md - fix a syntax left-over problem
  pathz.proto - remove an errant empty comment line.

**Review by marcushines — APPROVED** [review]

**Pull request merged**